### DISCUSSION TITLE
JD Builder

### DISCUSSION THREAD

**Discussion by darekmer** [templaza/astroid-framework#379] [General]
The guy wants to revive JD Builder
https://www.facebook.com/groups/246796345390331/permalink/9800225506713986/

**Comment by Chacapamac** [comment]
To my recollection, JD Builder was not a Joomla friendly builder at all. I make a total  review of ALL Joomla builder and if I remember well , I drop JD Builder even as a contender. To bad to even waist my time.

**Sadly, most big boys “Builder” didn’t even Pass my Minimum Criteria at that time...**
Almost 10000 views post in Joomla Forum —> https://forum.joomla.org/viewtopic.php?f=575&t=988600

**My only 2 necessary criteria for a great Joomla Page Builder Editor was:**

1. TOTAL INTEGRATION with the native Joomla system.
(articles manager, articles edition & custom modules edition, media's manager, categories...) and should present and HYPER IMPORTANT should have an IDENTICAL Joomla edition experience both from frontend & backend.
— This is important! Anything else will complicate the overall CMS experience to the detriment of Users/Editor and diminish the power of Joomla itself
— One Native Joomla content management & Edition system to learn
— Should work just like JCE or any other editors in complete tandem with Joomla FIRST (They can add gizmos if they want)
— Remember, we have to beat the competition (WP Gutemberg)

2. Being simple & intuitive to use and at the same time as powerful as a traditional editor like JCE.
— Also IDEALLY made by a Joomla multi developers renown establish business.

Any Page Builder that doesn’t comply with these 2 criteria is simply NOT “Joomla Friendly” and not using the full power of Joomla properly and detrimental to Joomla itself.

I’m still using my trusted JCE editor as you can create easily a complete WYSIWYG experience for Editors, including, Easy use of templates, Templates Color Schemes, bank of ready-made formatting, use of Bootstrap columns, complete custom editor for each categories of Editors, can be use THE SAME WAY in Joomla Backend/Frontend, and so much more....
All that in a normal minimalist code (compare to the bulky fragile “Page Builder” most of the time proprietary code)

JCE is working to get a system of movable code block in the edition mode (super easy in the code mode) and even now you can easily  copy, cut, paste block element (div, p, whatever) in the edition mode.

JCE is work
